docs/practice/cases.en-US.md

```diff
@@ -0,0 +1,86 @@
+---
+order: 0
+title: Cases
+---
+
+Ant Design is an implementation of UI Design Language。
+
+Starting in April 2015, Ant Design product rapidly expanded, covering multiple business lines and more than 40 systems.
```

Comment:
Starting in April 2015, more and more products of Ant Financial follow Ant Design specification, ....